### PR TITLE
NEW: pygwalker

### PR DESCRIPTION
**Pull request by gabrielcnr**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gabrielcnr** [comment]
@marcelotrevisani can you please add this package to conda-forge? Thank you

File: recipes/pygwalker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --ignore-installed
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5,<4.0
+    - pip
+    - poetry
```
Comment (marcelotrevisani) [inline recipes/pygwalker/meta.yaml:21]:
```suggestion
    - poetry-core
```

File: recipes/pygwalker/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  sha256: befae03ed94ac40c66ab87283c8d4ee567598d444c0ea375c3dd8dccb686f65a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --ignore-installed
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5,<4.0
```
Comment (marcelotrevisani) [inline recipes/pygwalker/meta.yaml:19]:
```suggestion
    - python >=3.6,<4.0
```

File: recipes/pygwalker/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    - pip
+    - poetry
+  run:
+    - python >=3.5,<4.0
```
Comment (marcelotrevisani) [inline recipes/pygwalker/meta.yaml:23]:
```suggestion
    - python >=3.6,<4.0
```

**Review by marcelotrevisani — APPROVED** [review]

**Pull request merged**